### PR TITLE
Speedup forward pass by more than 10x

### PR DESCRIPTION
**Pull request by melgor**
Change for loop by single-line command. Works for both float and cuda.

### CODE REVIEW

File: TripletEmbedding.lua [hunk 1]
```diff
@@ -18,11 +18,7 @@ function TripletEmbeddingCriterion:updateOutput(input)
    local p = input[2] -- positive
    local n = input[3] -- negative
    local N = a:size(1)
-   self.Li:resize(N)
-   for i = 1, N do
-      self.Li[i] = math.max(0, (a[i]-p[i])*(a[i]-p[i])+self.alpha-(a[i]-n[i])*(a[i]-n[i]))
-      --print(self.Li[i])
-   end
+   self.Li,_ = torch.max(torch.cat(torch.Tensor(N):zero():type(torch.type(a)) , (a - p):norm(2,2):pow(2) -  (a - n):norm(2,2):pow(2) + self.alpha, 2), 2)
```
Comment (Atcold) [inline TripletEmbedding.lua]: Could you remove the `,_` and force-push?

**Comment by Atcold** [comment]
Thank you. Please, check my comment and force-push. Then I'll merge it.

**Comment by Atcold** [comment]
Thank you! Before merging, could you name your commit `DEV - <description>` and remove the `,_`?

**Comment by melgor** [comment]
Done.

**Comment by Atcold** [comment]
Thank you, @melgor!

**Pull request merged**